webpack/scenes/ContentViews/Details/Repositories/SelectToggleHelper.js

```diff
@@ -0,0 +1,20 @@
+const onRepoSelect = (rows, setRows, setBulkActionDisabled) => (_event, isSelected, rowId) => {
+  let newRows;
+
+  if (rowId === -1) {
+    newRows = rows.map(row => ({ ...row, selected: isSelected }));
+  } else {
+    newRows = [...rows];
+    newRows[rowId].selected = isSelected;
+  }
+
+  if (newRows.some(row => row.selected)) {
+    setBulkActionDisabled(false);
+  } else {
+    setBulkActionDisabled(true);
+  }
```

Comment:
I think one thing that would help is if you moved this bulk action logic to an effect hook in the actual component:

```javascript
useEffect(() => {
    const rowsAreSelected = rows.some(row => row.selected)
    setBulkActionDisabled(!rowsAreSelected)
}, [JSON.parse(rows)]
```
This should keep up with the statuses switching if I'm reading things right! Also, `bulkActionEnabled` would be a little more readable both here and in the component IMO, double negatives tend to hurt my brain :brain: